smac/utils/io/cmd_reader.py

```diff
@@ -546,7 +552,29 @@ def _add_smac_options(self):
                default=0.5, type=float,
                help="[dev] probablity to run a random configuration"
                " instead of configuration optimized on the acquisition function")
-
+        ## Successive Halving
+        smac_opts.add_argument("--sh-min-budget", "--sh_min_budget",
+                dest='sh_min_budget',
+                default=None, type=int,
+                help="[dev] Set minimum budget for successive halving. Default is 1 (min. instances in 1st run)")
+        smac_opts.add_argument("--sh-max-budget", "--sh_max_budget",
+               dest='sh_max_budget',
+               default=None, type=int,
+               help="[dev] Set maximum budget for successive halving. Default is total instances")
```

Comment:
Maybe if you can't set the default value here you could change the help text to something like: "If not specified, i.e. None; all instances are used"